configs/sst_networking-firewall-unwanted.yaml

```diff
@@ -10,8 +10,12 @@ data:
   - c9s
 
   unwanted_packages:
+  - arptables-legacy
+  - arptables-services
```

Comment:
Does the `iptables-nft-services` package have a `Provides: arptables-services` ? If so, then we shouldn't put this in the unwanted list.

Reply:
OK, then I'll drop {arp,eb}tables-services from unwanted list.